jsonnetlib/k8s.jsonnet

```diff
@@ -304,6 +304,8 @@ cfg {
     envmap={},
     container={},
     volumes=[],
+    volumeMounts=[],
```

Comment:
you can use [`container.volumeMounts`](https://github.com/theplant/plantbuild/blob/master/example/k8s/tests/test_volumes.jsonnet) for this